docs/supported-array-libraries.md

```diff
@@ -131,8 +124,6 @@ For `linalg`, several methods are missing, for example:
 - `matrix_rank`
 Other methods may only be partially implemented or return incorrect results at times.
 
-The minimum supported Dask version is 2023.12.0.
```

Comment:
Do not advertise what you don't test. Also helplessly obsolete; we know for sure that less than 2024.9.0 won't work.